post-hackathon/README.md

```diff
@@ -0,0 +1,23 @@
+# Post-Hackathon 
+Congrats you made it through the hackathon and you now have some working project! What should you do? Definitely DO NOT just make your project a history. Here are the steps that you should follow to make your experience worth your effort. 
+
+## 1. Post-Mortem
+
+### The Positives
+Make a list of everything that went right in this project so that it can be repeated next time. From the small-scale accomplishments to the large-scale accomplishments, each detail counts and is probably worth noting! This makes it easier next time when you want to add to the project, develop a similar project with more functionalities, or use it as reference in an internship, job, and/or another experience of applying the knowledge.
```

Comment:
```suggestion
Make a list of everything that went right in this project so that it can be repeated next time. From the small-scale accomplishments to the large-scale accomplishments, each detail counts and is probably worth noting! This makes it easier next time when you want to add to the project, develop a similar project with more functionalities, or use it as reference in any kind of interview.
```